.npm/new.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+const packageJson = require("./package");
+const fs = require("fs");
+const { download } = require("./download");
+const { execute } = require("./execute");
+const path = require("path");
+
+(async () => {
+  let args = process.argv;
+  args.shift(); // node
+  const dirPath = path.dirname(args[0]);
+  args[0] = "new"; // Replace ".../cca.js" with "new"
+
+  const ccaVersion = /^\d\.\d\.\d/.exec(packageJson.version)[0];
```

Comment:
What is the reason for using a regex here? The version number in the package.json is exactly `0.6.0`. Is that not sufficient?

Reply:
For this exact kind of PR I will do manual publication on npm with RC versions so that I can test the PR work before doing a full create-comit-app release. Hence the package.json will be `0.6.0rc1`, hence we need to extract.

Reply:
Makes perfect sense :)
Do you mind documenting that in a commend / function? It was not obvious to me!